.travis.yml

```diff
@@ -10,6 +10,8 @@ jobs:
   include:
   - name: "Testing ansible playbook `deploy-discourse.yml`"
     env: TEST_ROLE=discourse
+  - name: "Testing ansible playbook `deploy-matterbridge.yml`"
```

Comment:
So this will be printed out in the log by Travis, right?

Reply:
It is actually the name of the job listed here:
https://github.com/PrivacyLx/devops/pull/31/checks?check_run_id=558053172